client/src/components/IndicatorsViz.tsx

```diff
@@ -312,10 +343,12 @@ class IndicatorsVizImplementation extends Component<IndicatorVizImplementationPr
     var dataMaximum = this.getDataMaximum();
     var suggestedYAxisMax =
       this.props.activeVis !== "hpdcomplaints" && this.props.activeVis !== "hpdviolations"
-        ? Math.max(
-            12,
-            Helpers.maxArray(this.groupData(timelineData.dobpermits.values.total) || [0]) * 1.25
-          )
+        ? this.props.activeVis !== "rentstabilizedunits"
+          ? Math.max(
+              12,
+              Helpers.maxArray(this.groupData(timelineData.dobpermits.values.total) || [0]) * 1.25
+            )
+          : unitsres + unitsres / 4
```

Comment:
gross looking but basically tried to add some space in the y-axis to accommodate "Sold to current owner" label